src/rules/transition/out.jl

```diff
@@ -10,6 +10,7 @@ end
 end
 
 @rule Transition(:out, Marginalisation) (m_in::DiscreteNonParametric, q_a::PointMass, meta::Any) = begin
+    @logscale 0
```

Comment:
`@call_rule` should use the `getaddons` function. See for example: https://github.com/biaslab/ReactiveMP.jl/blob/e83dd3330292363ec7d356d8605ad77d13d24359/src/rules/multiplication/out.jl#L20